src/client/src/rt-platforms/glue/reactive-trader.json

```diff
@@ -1,8 +1,8 @@
 [
   {
-    "title": "Reactive Trader",
+    "title": "Reactive Trader™",
     "type": "window",
-    "name": "reactive-trader",
+    "name": "reactive-trader-™",
```

Comment:
These `name`s are identifiers I think, they don't need the TM unless they will actually be displayed

Reply:
Okay, just removed TM from all the "name" fields.